.github/pull_request_template.md

```diff
@@ -1,6 +1,10 @@
-Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+📌 Review Process TL;DR:
+- ⏱️ We aim to review reasonably-sized PRs within 1 business day.
```

Comment:
I'm not sure if we should commit to this, or at least refactor this to define that we'll acknowledge any PR within 1 business day.